maven-core/src/test/java/org/apache/maven/project/ProjectBuilderTest.java

```diff
@@ -332,8 +320,8 @@ public void testBuildProperties()
         ProjectBuildingRequest configuration = new DefaultProjectBuildingRequest();
         configuration.setRepositorySession( mavenSession.getRepositorySession() );
         configuration.setResolveDependencies( true );
-        List<ProjectBuildingResult> result = projectBuilder.build( Collections.singletonList( file ), true, configuration );
-        MavenProject project = result.get( 0 ).getProject();
+        List<ProjectBuildingResult> result = projectBuilder.build( Collections.singletonList(file), true, configuration );
+        MavenProject project = result.get(0).getProject();
```

Comment:
Where did the spaces go?

Reply:
Same here, I restored the file, will fix it in a separate commit